ansible/roles/sonarqube/files/health_check.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+# Health Check for Sonarqube
+
+# Configure descriptors for logging
+exec 1> >(logger -s -t $(basename $0)) 2>&1
+
+# Functions
+# Fail health check and update auto scaling group
+fail_health () {
+    echo "health check failed"
+    aws --region eu-west-2 autoscaling set-instance-health --instance-id $INSTANCE_ID --health-status Unhealthy
+    exit 1
+}
+
+# Succeed health check and update autoscaling group
+succeed_health () {
+    echo "health check successful"
+    aws --region eu-west-2 autoscaling set-instance-health --instance-id $INSTANCE_ID --health-status Healthy
+    exit 0
+}
+
+# Main
+# Check if the required packages are installed
```

Comment:
I would keep this script focussed on doing the health check and just ensure that the packages are installed elsewhere. This role is coupled with the AMI after all

Reply:
As the output of the script is put to syslog via logger this helps do diagnose issues with the health check if it is not running. So would prefer to keep it in if possible as then you can directly see when the script is not running due to dependencies.